src/utils/webrtc/simplewebrtc/localmedia.js

```diff
@@ -132,6 +133,38 @@ LocalMedia.prototype.isLocalMediaActive = function() {
 	return this._localMediaActive
 }
 
+/**
+ * Adjusts video constraints to work around bug in Chromium.
+ *
+ * In Chromium it is not possible to increase the resolution of a track once it
+ * has been cloned, so the track needs to be initialized with a high resolution
+ * (otherwise real devices are initialized with a resolution around 640x480).
+ * Therefore, the video is requested with a loose constraint for a high
+ * resolution, so if the camera does not have such resolution it will still
+ * return the highest resolution available without failing.
+ *
+ * @param {Object} constraints the constraints to be adjusted
+ */
+LocalMedia.prototype._adjustVideoConstraintsForChromium = function(constraints) {
+	const parser = new UAParser()
+	const browserName = parser.getBrowser().name
+
+	if (browserName !== 'Chrome' && browserName !== 'Chromium') {
```

Comment:
I guess this should also match for safari, edge and opera.
But it does not. I added:
```js
OC.Notification.show(browserName)
```

And it prints "Safari"
Also visible in https://github.com/nextcloud/spreed/blob/1795852d293de3cc3f6131e6fde259ce4c9aa586/src/mixins/browserCheck.js#L45-L62

So I guess we just extend the list to include all those?